src/sections/TrustedBy/TrustedBy.tsx

```diff
@@ -55,7 +56,7 @@ const CompanyImage = styled.img`
 export const TrustedBy = () => (
   <Container>
     <Content>
-      <div>Solution trusted by:</div>
+      <div style={{ width: 150 }}>Solution trusted by:</div>
```

Comment:
Why not `styled.div`?

Reply:
Same reason as in https://github.com/vazco/forminer/pull/61#discussion_r1285029816, but I will change it